.github/pull_request_template.md

```diff
@@ -1 +1,7 @@
-> See JIRA https://skedify.atlassian.net/browse/SKED-
\ No newline at end of file
+> See JIRA https://skedify.atlassian.net/browse/SKED-
+
+<!--  
+In an effort to increase security awareness we want to remind ourselves to pay attention to security when writing code.
+👉 Mark this checkbox if you payed attention to security.
+-->
+- [ ] Security Awareness Checkbox 🛡️
```

Comment:
This will become "how to test" scenario very fast 😂

Reply:
```suggestion
- [x] Security Awareness Checkbox 🛡️
```

Saved you a click 😘